cmd/jaeger/badger_config.yaml

```diff
@@ -0,0 +1,43 @@
+service:
+  extensions: [jaeger_storage]
+  pipelines:
+    traces:
+      receivers: [otlp, jaeger, zipkin]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter]
+
+extensions:
+  jaeger_query:
+    trace_storage: memstore
+    ui_config: ./config-ui.json
+
+  jaeger_storage:
+    badger:
+      memstore:
+        directory_key: "/tmp/jaeger/"
+        directory_value: "/tmp/jaeger/"
+        ephemeral: false
+        maintenance_interval: 5
+        metrics_update_interval: 10
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+  jaeger:
+    protocols:
+      grpc:
+      thrift_binary:
+      thrift_compact:
+      thrift_http:
+
+  zipkin:
```

Comment:
I'd say for illustration purposes of badger storage these are not needed

Reply:
removed them